combat-trainer.lic

```diff
@@ -3449,6 +3472,27 @@ class GameState
     true
   end
 
+  def update_analyze_array_combo
+    ret = false
+    if Flags["ct-accuracy-ready"] && @barb_accuracy
```

Comment:
❓ If, for example, two analyze types are ready, won't this try both?

Reply:
if two analyzes are ready it will indeed do both and it wont repeat any analyze combos until one of the flags is ready again

currently it will run through each analyze repeatedly every time one is ready and if none are ready it will nonstop analyze flame